driver.sh

```diff
@@ -9,22 +9,50 @@ check_dependencies() {
   command -v gcc
   command -v aarch64-linux-gnu-as
   command -v aarch64-linux-gnu-ld
+  command -v arm-linux-gnueabi-as
+  command -v arm-linux-gnueabi-ld
   command -v qemu-system-aarch64
+  command -v qemu-system-arm
   command -v timeout
   command -v unbuffer
   command -v clang-8
 
   set +e
 }
 
-parse_parameters() {
+setup_variables() {
   while [[ $# -ge 1 ]]; do
     case $1 in
       "-c"|"--clean") cleanup=true ;;
     esac
 
     shift
   done
+
+  # arm64 is the current default if nothing is specified
+  [[ -z "${ARCH:-}" ]] && ARCH=arm64
+  export ARCH
+  case $ARCH in
+    "arm")
+      config=multi_v7_defconfig
+      image_name=zImage
+      qemu="qemu-system-arm"
+      qemu_cmdline=( -initrd images/arm/rootfs.cpio
```

Comment:
It's curious to me that we use a rootfs for aarch64, but an initrd for arm32.  Maybe it doesn't matter, but I was curious if you had strong beliefs on one vs the other @nathanchance ?

Reply:
Ugh I forgot to try to get an arm32 rootfs working properly, I'll do that tonight. I just pushed what I knew worked.

Reply:
Alright I got it to work locally, let's see if CI agrees then I can rebase and squash then I think this should be good to go!